src/main.cpp

```diff
@@ -174,6 +174,64 @@ std::string printVDeskDispatch(eHyprCtlOutputFormat format, std::string arg) {
     return "";
 }
 
+std::string printVDesksDispatch(eHyprCtlOutputFormat format, std::string arg) {
+    std::string out;
+    if (format == eHyprCtlOutputFormat::FORMAT_NORMAL) {
+        out += "Virtual desks\n";
+        int index = 0;
+        for(auto const& [vdeskId, desk] : manager->vdesksMap) {
+            unsigned int windows = 0;
+            std::string workspaces;
+            bool first = true;
+            for(auto const& layout : desk->layouts) {
```

Comment:
This should only be done for the current, active layout. No need to iterate over all of them

Reply:
Updated the code to only use the active layout for each desk